napalm_panos/panos.py

```diff
@@ -509,28 +524,26 @@ def get_interfaces(self):
                 interface_info_xml = xmltodict.parse(self.device.xml_root())
                 interface_info_json = json.dumps(interface_info_xml['response']['result']['hw'])
                 interface_info = json.loads(interface_info_json)
-            except AttributeError:
-                interface_info = {}
+            except KeyError:
+                # loopback sub-ifs don't return a 'hw' key
+                interface_dict[intf] = LOOPBACK_SUBIF_DEFAULTS
+                continue
 
-            name = interface_info.get('name')
-            state = interface_info.get('state')
+            state = interface_info.get('state') == 'up'
```

Comment:
my personal preference is to be explicit, e.g. 
```
state = False
if interface_info.get('state') == 'up':
    state = True
```

but won't lose sleep either way.